src/Stats/Persister/RedisPersister.php

```diff
@@ -76,6 +81,7 @@ public function incrementRepositoryAccessType(string $repository, string $type):
         return $this;
     }
 
+    // @todo $maxListLength seems to be useless, remove in next major version?
```

Comment:
I know. This part of the codebase it's much legacy... When we migrate to the version 3 we have done copy and paste! I believe there are many unused methods. :)

Reply:
Ok, so, what to do here?

Reply:
let's clean it up and remove it! ;)